src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,403 @@
+package duke.task;
+
+import duke.exception.*;
+import duke.ui.*;
```

Comment:
As mentioned above, I think the imported classes should be listed explicitly as per the coding standard given?
```suggestion
import duke.exception.DukeException;
import duke.ui.Ui;
```